addons/property_panel/float_slider/float_slider.gd

```diff
@@ -18,6 +18,8 @@ signal value_changed(value: float)
 @export var step : float
 ## The sensitivity while dragging with the mouse to change the value.
 @export var sensitivity := 1000.0
+## The expression to evaluate to get the value.
+@export var expression := true
```

Comment:
This could be more expressive, maybe "allow_expressions". The documentation should be updated accordingly.